CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+<!-- omit in toc -->
+# Contributing to Capgo
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+### Running tests localy
+
+This project uses a custom test runner located in [tests_backend](https://github.com/Cap-go/capgo/tree/main/tests_backend).
+There exists some requirements to run the tests:
+ * Have minio running (see `tests_backend/gh_actions` for referance) For testing the CLI this is not enough, you have to set up the `MINIO_URL` variable. In linux you can use 
+ ```sh
+ export MINIO_URL=$(docker inspect minio1 | grep 'Gateway' | head -n 1 | sed -e 's/            "Gateway": "//g' | sed -e 's/",//g')
```

Comment:
perhaps, however please be aware that this is a linux only example. I have no idea how to run this in powershell and making this into a script could perhaps make this command compatible with both linux and windows. But I am worried will happen if this script fails

Reply:
but do we need to take care of windows also?

all of the OSS I've contributed only works on Unix systems; this is pretty standard, and the provided code already works only on Linux. make it a script that'd make it easy to run (instead of copying/pasting, it'd be just run the script).

although I think this would make easier to execute these stuff to newcomers, it's optional, so feel free to solve this issue as you want